src/data/python-quiz.ts

```diff
@@ -1672,12 +1672,14 @@ const pythonQuiz = [
   {
     Question:
       "In Python, which of the following dunder methods defines the behavior of the addition operator + for objects of a class?",
+
     Answer: "__add__",
     Distractor1: "__len__",
     Distractor2: "__sub__",
     Distractor3: "__mul__",
     Explanation:
       "Dunder methods (Double underscore) are special methods reserved in Python which affect's the behavior of the object and the interaction of the object with other objects.",
+
```

Comment:
this space doesn't need to be here
please remove

Reply:
Will change it too.